src/udev/udev-node.c

```diff
@@ -334,37 +370,6 @@ size_t udev_node_escape_path(const char *src, char *dest, size_t size) {
         return size - 1;
 }
 
-static int stack_directory_get_name(const char *slink, char **ret) {
```

Comment:
Please do not move this up, but move `stack_directory_open()` down. I.e. I'd like to keep `udev_node_escape_path()` near `stack_directory_get_name()`.

Reply:
I also moved `stack_directory_lock()` down to keep it next to `stack_directory_open()`